scripts/templates/sonatype.com_v1alpha1_nexusrepo_cr.yaml

```diff
@@ -9,35 +9,35 @@ spec:
   ingress:
     additionalRules: null
     annotations: null
-    defaultRule: "false"
+    defaultRule: false
     dockerIngress:
       annotations: null
-      enabled: "false"
+      enabled: false
       host: example.com
       name: nexus-docker-ingress
     dockerSubdomain: false
-    enabled: "false"
+    enabled: false
     host: example.com
     name: nexus-ingress
     tls:
-      enabled: "false"
+      enabled: false
       secretName: tlsSecretName
   license:
     fileContentsBase64: your_license_file_contents_in_base_64
     secretName: nexus-repo-license
   nexusData:
     local:
       accessMode: ReadWriteOnce
-      enabled: "true"
+      enabled: true
       pv:
         matchNodes: example-node-1 example-node-2 example-node-3
         name: nexusrepo-local-storage-pv
-        size: 10Gi
+        size: 2Gi
```

Comment:
is it an intentional change? I see per our [official docs](https://help.sonatype.com/display/NXRM3M/Sonatype+Nexus+Repository+System+Requirements#SonatypeNexusRepositorySystemRequirements-DiskSpace): "When available disk space drops below 4GB the database will switch to read-only mode."

Reply:
This is just for storing logs i.e. not database.